controllers.js

```diff
@@ -23,4 +23,13 @@ exports.fetchArticleById = (req, res, next) => {
     }).catch(err => {
         next(err)
     })
+}
+
+exports.fetchCommentsById = (req, res, next) => {
+    const id = req.params.article_id;
+    getCommentsById(id).then((results) => {
+        res.status(200).send(results)
```

Comment:
can we pop the results on an object with a key that describes what's coming back